### PR TITLE
UF-320: Improvements on admin page and preferences view

### PR DESCRIPTION
**Pull request by paulovmr**
* Default screen for Admin Page.
* Load/save with custom scope resolution strategy.
* Making empty preferences not selectable on the tree.
* "Go back" button on admin perspective (will probably be replaced for Uberfire breadcrumbs in the future).

### CODE REVIEW

**Comment by paulovmr** [comment]
Part of an ensemble:

* https://github.com/uberfire/uberfire/pull/558
* https://github.com/droolsjbpm/guvnor/pull/386
* https://github.com/droolsjbpm/kie-wb-common/pull/558
* https://github.com/droolsjbpm/kie-wb-distributions/pull/404

File: uberfire-extensions/uberfire-preferences/uberfire-preferences-client/src/main/java/org/uberfire/ext/preferences/client/admin/AdminPagePerspective.java [hunk 1]
```diff
@@ -62,4 +74,22 @@ void configurePerspective( final PlaceRequest placeRequest ) {
         perspective.getRoot().addPart( new PartDefinitionImpl( new DefaultPlaceRequest( AdminPagePresenter.IDENTIFIER,
                                                                                         placeRequest.getParameters() ) ) );
     }
+
+    @WorkbenchMenu
+    public Menus getMenus() {
+        if ( perspectiveIdentifierToGoBackTo != null ) {
+            return MenuFactory
+                    .newTopLevelMenu( "Go to the previous page" )
```
Comment (manstis) [inline uberfire-extensions/uberfire-preferences/uberfire-preferences-client/src/main/java/org/uberfire/ext/preferences/client/admin/AdminPagePerspective.java]: i18n

File: uberfire-extensions/uberfire-preferences/uberfire-preferences-client/src/main/java/org/uberfire/ext/preferences/client/central/PreferencesCentralPerspective.java [hunk 2]
```diff
@@ -60,50 +61,31 @@ public void perspectiveChangeEvent( @Observes UberfireDockReadyEvent dockReadyEv
...
+    @PostConstruct
+    public void setupNavBarDock() {
         dock = new UberfireDock( UberfireDockPosition.WEST,
                                  "ADJUST",
-                                 placeRequest,
+                                 new DefaultPlaceRequest( PreferencesCentralNavBarScreen.IDENTIFIER ),
                                  IDENTIFIER )
-                .withSize( 420 )
-                .withLabel( title );
+                .withLabel( "Preferences" )
```
Comment (manstis) [inline uberfire-extensions/uberfire-preferences/uberfire-preferences-client/src/main/java/org/uberfire/ext/preferences/client/central/PreferencesCentralPerspective.java]: i18n?

**Comment by csadilek** [comment]
+1

File: uberfire-extensions/uberfire-preferences/uberfire-preferences-client/src/main/java/org/uberfire/ext/preferences/client/admin/AdminPagePerspective.java (hunk 1 above)
Comment (paulovmr) [inline uberfire-extensions/uberfire-preferences/uberfire-preferences-client/src/main/java/org/uberfire/ext/preferences/client/admin/AdminPagePerspective.java]: Done

File: uberfire-extensions/uberfire-preferences/uberfire-preferences-client/src/main/java/org/uberfire/ext/preferences/client/central/PreferencesCentralPerspective.java (hunk 2 above)
Comment (paulovmr) [inline uberfire-extensions/uberfire-preferences/uberfire-preferences-client/src/main/java/org/uberfire/ext/preferences/client/central/PreferencesCentralPerspective.java]: Done

**Comment by ederign** [comment]
@paulovmr this is a isolated PR? or have child's?

**Comment by manstis** [comment]
@ederign @paulovmr All related PRs reviewed and approved.. This PR needs to "go first".. please ping me on IRC when you're done and I'll do the others.

**Comment by paulovmr** [comment]
jenkins retest this

**Comment by porcelli** [comment]
jenkins retest this

**Pull request merged**